recipe/meta.yaml

```diff
@@ -11,18 +11,18 @@ source:
 
 build:
   number: 1
-  skip: True  # [win]
+  skip: True  # [win or py<35]
   script: python -m pip install --no-deps --ignore-installed .
 
 requirements:
   build:
     - {{ compiler('c') }}
   host:
-    - python >=3.5
```

Comment:
This is fine for noarch but for compiled code you want it to build on every python version in the matrix and not only what satisfy `>=`, Python 3.7 in this case.